src/_includes/website_navigation.liquid

```diff
@@ -12,7 +12,7 @@
 
     <!-- Logo -->
     <a href="{{ '/' | prepend: site.baseurl }}" class="navbar-brand g-mr-25--lg g-mr-50--xl">
-      <img src="{{ '/assets/custom/img/codurance.png' | prepend: site.baseurl }}" class="header-logo" alt="Codurance Logo">
```

Comment:
Are we still using `/assets/custom/img/codurance.png` anywhere? If not let's delete :)

Reply:
We have deleted the `/assets/custom/img/codurance.png` but we still have one in `/assets/img/codurance.png` its not being used anywhere but there's a note attached to it, and we weren't sure if it was ok to delete it.